monero-outreach-docs/translations/pt-br/website-sections/about_pt_br.md

```diff
@@ -0,0 +1,60 @@
+# Sobre Monero Outreach
+
+Um grupo independente
+
+Nós somos um time de criadores, escritores, técnicos, tradutores e visionários. Monero Outreach começou em Junho de 2018 e continua ganhando novos e talentosos contribuidores. Nós somos um grupo de trabalho da comunidade descentralizada do Monero e, desta forma, não reivindicamos nenhum nível de autoridade sobre o Monero - nós estamos aqui simplesmente para seguir o mandato que estipulamos para nós mesmos, porque nós amamos o Monero.
+
+## Mandato Monero Outreach
+
+1. Impulsionar uma maior adoção e conhecimento através de uma variedade de atividades relacionadas à divulgação - particularmente relações públicas, educação e marketing.
+2. Respeitar o trabalho feito pelos Desenvolvedores Core e pelos grupos de trabalho da comunidade, os princípios fundamentais do Monero e o espírito Monero, em todos os sentidos, para que nós não prejudiquemos a integridade do que do que torna o Monero excelente.
```

Comment:
"do que" is doubled. My correction would be this:

2. Respeitar o trabalho feito pelos Desenvolvedores Core e pelos grupos de trabalho da comunidade, os princípios fundamentais do Monero e o espírito Monero, em todos os sentidos, para que nós não prejudiquemos a integridade do que torna o Monero excelente.